tests/test_base_plugin.py

```
@@ -1554,11 +1559,48 @@ def get_matrix_commands(self):
             result = plugin.get_matching_matrix_command(mock_event)
             self.assertEqual(result, "cmd2")
 
+    def test_get_matching_matrix_command_with_args_matches_formatted_mention_pill(self):
+        """get_matching_matrix_command_with_args should parse command+args from formatted mention pills."""
+
+        class MultiCmdPlugin(BasePlugin):
+            plugin_name = "multi"
+            is_core_plugin = True
+
+            async def handle_meshtastic_message(
+                self, packet, formatted_message, longname, meshnet_name
```

Ruff (0.15.10)

[warning] Unused method argument: `packet`

(ARG002)